zebra-chain/src/block.rs

```diff
@@ -116,6 +119,30 @@ impl Block {
 
         Ok(())
     }
+
+    /// Access the sprout::Nullifiers from all transactions in this block.
+    pub fn sprout_nullifiers(&self) -> impl Iterator<Item = &sprout::Nullifier> {
+        self.transactions
+            .iter()
+            .map(|transaction| transaction.sprout_nullifiers())
+            .flatten()
+    }
+
+    /// Access the sapling::Nullifiers from all transactions in this block.
```

Comment:
Nit: Documentation link.

```suggestion
    /// Access the [`sapling::Nullifier`]s from all transactions in this block.
```